.github/ISSUE_TEMPLATE/3_feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
```

Comment:
Please capitalize the header row of the table: name, about, etc.

Please add a period to the end of full sentences: Suggest an idea...

Reply:
Done.